server/odc-service/src/main/java/com/oceanbase/odc/service/task/TaskService.java

```diff
@@ -275,8 +281,19 @@ public void start(@NonNull Long id, Object taskResult) {
     @Transactional(rollbackFor = Exception.class)
     public void succeed(Long id, Object taskResult) {
         TaskEntity taskEntity = nullSafeFindById(id);
-        taskEntity.setStatus(TaskStatus.DONE);
-        taskEntity.setProgressPercentage(100);
+        if (taskEntity.getTaskType() == TaskType.MULTIPLE_ASYNC) {
```

Comment:
restore this original logic and move this into `MultipleDatabaseChangeRuntimeFlowableTask`